ts/state/selectors/conversations.ts

```diff
@@ -1174,3 +1178,67 @@ export const getIsSelectedConvoInitialLoadingInProgress = createSelector(
   getSelectedConversation,
   (convo: ReduxConversationType | undefined): boolean => Boolean(convo?.isInitialFetchingInProgress)
 );
+
+export const getSelectedConversationExpirationModes = createSelector(
+  getSelectedConversation,
+  (convo: ReduxConversationType | undefined) => {
+    let modes = DisappearingMessageConversationSetting;
+    // TODO legacy messages support will be removed in a future release
+    // TODO remove legacy mode
+    modes = modes.slice(0, -1);
```

Comment:
The conversation setting is a hardcoded array 
`export const DisappearingMessageConversationSetting = ['off', ...DisappearingMessageMode, 'legacy'];` 

We could do an indexOf and delete but since we will remove this code a few weeks after the release...